yaml-tests/src/main/java/com/apple/foundationdb/relational/yamltests/YamlConnection.java

```diff
@@ -108,4 +108,11 @@ public interface YamlConnection extends AutoCloseable {
     SemanticVersion getInitialVersion();
 
     <T> T executeTransactionally(SQLFunction<YamlConnection, T> transactionalWork) throws SQLException, RelationalException;
+
+    /**
+     * The Cluster File that this connection is connected to, so that other connections can point to the same
```

Comment:
I'm not super committed to this API, but at some point we'll want multi-cluster testing in yaml-tests, and I think that's a better time to really think about it.